build/mathquill.d.ts

```diff
@@ -22,6 +22,47 @@ declare namespace MathQuill {
       handlers?: HandlerOptions;
     };
 
+    interface MQNode {
+      id: number;
+      parent: NodeBase;
+    }
+  
+    interface MQSelection {
+      id: number;
+      getEnd(dir: number): number;
+    }
+  
+    interface NodeBase extends MQNode {
+      ctrlSeq: string | undefined;
+      blocks: MQNode;
+    }
+  
+    interface Cursor {
+      parent: MQNode;
+      selection: MQSelection | undefined;
+  
+      show(): Cursor;
+      hide(): Cursor;
+      insAtRightEnd(root: ControllerRoot): Cursor;
+      insRightOf(el: MQNode): Cursor;
+      insLeftOf(el: MQNode): Cursor;
+      startSelection(): void;
+    }
+  
+    interface Controller {
+      parent: string;
+      root: ControllerRoot;
+      cursor: Cursor;
+  
+      backspace(): Controller;
+      seek(targetElm, clientX, _clientY): Controller;
+    }
+  
+    interface ControllerRoot {
+      controller: Controller;
+      cursor?: Cursor;
+    }
+
```

Comment:
**question**: I put these inside the `v3` namespace. Should they be outside, maybe around line 9 instead?